core/types/src/key_request.rs

```diff
@@ -166,6 +166,25 @@ impl KeyRequestBuilder {
     }
 }
 
+pub struct Key128bit(sgx_key_128bit_t);
```

Comment:
This is in `key_request.rs` it may be that the file should be renamed to `key.rs` or similar
The request gets a `Key128bit` so the two are pretty closley tied, but i could also see that the request and the key each get their own module.